src/view3d.cpp

```diff
@@ -311,37 +461,42 @@ void View3D::paintGL()
 //        s[barSize] = '|';
 //    tfm::printfln("%12f %4d %s", quality, frameTime, s);
 
-    m_incrementalFramebuffer->release();
-    QGLFramebufferObject::blitFramebuffer(0, QRect(0,0,width(),height()),
-                                          m_incrementalFramebuffer.get(),
-                                          QRect(0,0,width(),height()),
-                                          GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    // TODO: this should really render a texture onto a quad and not use glBlitFramebuffer
```

Comment:
Right.  There's an extra complexity here - the depth should be copied as well as the colour.  How do you deal with that using a texture, without resorting to `gl_FragDepth`?